numpy/ma/core.py

```diff
@@ -8160,18 +8160,18 @@ def correlate(a, v, mode='valid', propagate_mask=True):
 
     >>> a = np.ma.array([1, 2, 3], mask=[False, True, False])
     >>> v = np.ma.array([0, 1, 0])
-    >>> np.ma.correlate(a, v, mode='valid', propagate_mask=True)
+    >>> np.ma.correlate(a, v)
     masked_array(data=[--],
                  mask=[ True],
            fill_value=999999,
                 dtype=int64)
 
-    Correlation with different modes:
+    Correlation with different modes and mixed elements:
```

Comment:
```suggestion
    Correlation with different modes and mixed array types:
```

Reply:
Just noticed I missed that. It's not elements, but array types.

Reply:
Looks like you missed adding this in. Once that gets done, I'll review this again and it looks like we can send it to Numpy as a PR. Ping me on the review task when this gets added in. https://github.com/possee-org/genai-numpy/issues/54